### ISSUE TITLE
It would be doable to replicate the old maff behaviour?

### ISSUE THREAD

**Issue by Carml82** [gildas-lormeau/SingleFileZ#89]
Hi, like many old users of Firefox I used for a long time the unbeatable Maff extension (still I use it via Waterfox), so I'm asking if it could be possible to use SinglefileZ exactly like the old Maff. I don't mind if it would necessary to install the addons in order to save and view them, as long as the format is (like is now SingleFileZ) a valid zip file. I found very useful Maff to save multiple pages at once and still I miss a lot its functionality.
Thanks in advance

**Comment by gildas-lormeau** [comment]
You can save multiple pages at once with SingleFileZ by selecting tabs. However, they will be saved as separate files.

**Comment by Carml82** [comment]
Yes I know that, what I'm asking is if would be too difficult for
SingleFileZ to create a single one containing all the selected tabs

Il dom 11 apr 2021, 19:22 Gildas ***@***.***> ha scritto:

**Comment by gildas-lormeau** [comment]
It represents some work but it's technically doable. I renamed the issue and tagged it as a feature request.

**Comment by Carml82** [comment]
Thanks in advance, also for the work you've done until now

Il lun 12 apr 2021, 23:45 Gildas ***@***.***> ha scritto:

**Comment by Carml82** [comment]
Hi I know it's a long time since I made the request,but there are there any updates on it? I've been trying to use Waterfox-Classic, Palemoon and Basilisk-Browser, but they're showing (more or less) more and more problems in staying in sync with the technology advancements and having the possibility again to have a single zip with tons of tabs on Firefox would be a nice to have commodity (like it was until Firefox 57). Thanks in advance.
